service/history/replication/task_processor.go

```diff
@@ -430,10 +430,12 @@ func (p *taskProcessorImpl) processSingleTask(replicationTask *types.Replication
 			tag.Error(err),
 		)
 		//TODO: uncomment this when the execution fixer workflow is ready
-		//if err = p.triggerDataInconsistencyScan(replicationTask); err != nil {
-		//	p.logger.Warn("Failed to trigger data scan", tag.Error(err))
-		//	p.metricsClient.IncCounter(metrics.ReplicationDLQStatsScope, metrics.ReplicationDLQValidationFailed)
-		//}
+		// Uncommenting this to verify that the scanner indeed gets triggered.
+		if err = p.triggerDataInconsistencyScan(replicationTask); err != nil {
+			p.logger.Info("Triggering a data scan")
```

Comment:
the info logs should be removed, because they don't help with debugging and also uses lots of quota

Reply:
Added these to test the stuff on staging. Will remove afterwards.